### ISSUE TITLE
Support scala

### ISSUE THREAD

**Issue by applideveloper** [Azure/draft#393]
If draft support Scala as well, draft will be more Awesome.

**Comment by bacongobbler** [comment]
Yes! We're focusing on other efforts at this time, but we plan to have more language packs available, including Scala. Please feel free to write up a pack for Scala if you feel up to the task! :)

**Comment by michelleN** [comment]
@applideveloper Here is some more information on creating packs: https://github.com/Azure/draft/blob/master/docs/packs.md

**Comment by ganesh47** [comment]
@michelleN Did you mean this https://github.com/Azure/draft/blob/master/packs/README.md ?
I can see #287  being discussed too and can see some details on https://github.com/Azure/draft/blob/master/docs/reference/dep-003.md as well!


But would like to understand better on how we could create custom packs for language-ecosystem like scala-sbt ?
Any reference for newbies ?

I would like to create a custom pack for scala (sbt-akka ecosystem)

**Comment by bacongobbler** [comment]
[DEP 003: Draft Pack Format](https://github.com/Azure/draft/blob/master/docs/reference/dep-003.md) goes into detail on how to create custom language packs. :)
